spec/Services/Exporter/DomPdfWishlistExporterSpec.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace spec\BitBag\SyliusWishlistPlugin\Services\Exporter;
+
+use BitBag\SyliusWishlistPlugin\Factory\DomPdfFactoryInterface;
+use BitBag\SyliusWishlistPlugin\Model\VariantPdfModelInterface;
+use BitBag\SyliusWishlistPlugin\Services\Exporter\DomPdfWishlistExporter;
+use BitBag\SyliusWishlistPlugin\Services\Exporter\DomPdfWishlistExporterInterface;
+use Doctrine\Common\Collections\ArrayCollection;
+use Dompdf\Dompdf;
+use PhpSpec\ObjectBehavior;
+use Twig\Environment;
+
+final class DomPdfWishlistExporterSpec extends ObjectBehavior
+{
+    public function let(
+        Environment $twigEnvironment,
+        DomPdfFactoryInterface $domPdfFactory
+    ) {
+        $this->beConstructedWith(
+            $twigEnvironment,
+            $domPdfFactory
+        );
+    }
+
+    public function it_is_initializable(): void
+    {
+        $this->shouldHaveType(DomPdfWishlistExporter::class);
+        $this->shouldImplement(DomPdfWishlistExporterInterface::class);
+    }
+
+    public function it_returns_pdf_as_attachment(
+        Environment $twigEnvironment,
+        VariantPdfModelInterface $pdfModel,
+        Dompdf $dompdf,
```

Comment:
Do we have any interface for this class?

Reply:
Dompdf is external package without provided interface